dm/master/workerrpc/rawgrpc.go

```diff
@@ -31,15 +31,15 @@ import (
 type GRPCClient struct {
 	conn   *grpc.ClientConn
 	client pb.WorkerClient
-	closed int32
+	closed *atomic.Bool
```

Comment:
```suggestion
	closed atomic.Bool
```

Do not use pointer to be consistance with other declares.